### ISSUE TITLE
Pair devices without manual tapping on a popup

### ISSUE THREAD

**Issue by man4567890** [danielpaulus/go-ios#139]
**Describe the bug**
Not sure if it is a bug

**To Reproduce**
Plug in device and run ./go-ios pair

**Expected behavior**
The device is paired with the host no manual intervention required

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu
 
**Smartphone (please complete the following information):**
ProductName: iPhone OS
ProductType: iPhone9,3
ProductVersion: 15.4.1

**Additional context**
How would you run the ./go-ios pair command, I do not have physical access to the device and would like to pair the device using command line only, IE bypass the "tapping the trust message on the device", Is this possible, if so, could you please give me a procedure on how to do this?

**Comment by shamanec** [comment]
There are two steps you need to do to be able to pair devices without manually interacting:  
1. Supervise your devices. You can find the steps needed to do it [here](https://github.com/danielpaulus/go-ios/issues/134). After you finish the supervision setup you should have a *.p12 file and its password.  
2. Then you pair your supervised devices using the file and password from step 1 like this:  
```ios pair --p12file="supervision.p12" --password="${SUPERVISION_PASSWORD}"```

**Comment by shamanec** [comment]
Yes, you need to manage all the devices under the same profile to be able to pair them without the Trust popup. But I think it will keep working until you hard reset the devices(might even work after hard reset but I haven't tried)

**Comment by man4567890** [comment]
Thanks

**Comment by nanoscopic** [comment]
Make really really sure to backup the credentials when setting up supervision. If you lose those you can, for the most part, never "unsupervise" them.

**Comment by danielpaulus** [comment]
Closing during issue triage: this is a usage/support question or too broad to track as an active bug. If there is still a concrete issue on the latest go-ios release, please open a focused new issue or ask us to reopen this one with current version details, exact commands, and logs.
